src/App.js

```diff
@@ -1,5 +1,43 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  async run() {
+    let input = await Console.readLineAsync('덧셈할 문자열을 입력해 주세요.\n');
+
+    let answer = 999999;
+
+    if (input === "") {
+      answer = 0;
+    }else{
+      function extractCustomDelimiter(input) {
+        const DELIMITER_MATCH = input.match(/\/\/(.+?)\\n/);
+        if (DELIMITER_MATCH) {
+          return DELIMITER_MATCH[1];
+        }
+        return null;
+      }
+
+      let customDelimiter = extractCustomDelimiter(input);
+      let delimiters = [',', ':'];
+      
+      if (customDelimiter) {
+        input = input.split("\\n")[1];
```

Comment:
```suggestion
        input = input.split("\n")[1];
```
요렇게 작성해야 명확하게 줄바꿈이 분리될 것 같습니다!

Reply:
해당 부분은 줄바꿈을 위한 것이 아닌 커스텀 구분자를 정의할 때 사용자가 string 값을 입력하면서 `//(구분자)\n숫자(구분자)숫자` 형태에서 \n 자체가 string이기 때문에 이를 기준으로 실제 숫자 데이터만 추출하는 역할을 하고 있는 의미에서 작성했습니다! 줄바꿈으로 분리를 해야하는 걸까요??

Reply:
`"\\n"`으로 `split`하여야 개행 역할을 하는 특수문자를 기준으로 분리하는 것이 아닌 **string을 기준으로 분리하는 것**으로 인식하기에

저도 suggested change가 아닌 기존 코드가 맞는 것 같습니다!